planner/core/plan_cache_lru.go

```diff
@@ -183,8 +183,12 @@ func (l *LRUPlanCache) removeOldest() {
 
 // removeFromBucket remove element from bucket
 func (l *LRUPlanCache) removeFromBucket(element *list.Element) {
-	bucket := l.buckets[string(hack.String(element.Value.(*planCacheEntry).PlanKey.Hash()))]
+	hash := string(hack.String(element.Value.(*planCacheEntry).PlanKey.Hash()))
```

Comment:
How about creating a dedicated function like `strHashKey(k kvcache.Key, deepCopy bool)` to do this?